src/User32.Desktop/User32.cs

```diff
@@ -109,6 +128,12 @@ public static partial class User32
         [DllImport(nameof(User32))]
         public static extern int SendMessage(IntPtr hWnd, int wMsg, IntPtr wParam, IntPtr lParam);
 
+        [DllImport(nameof(User32))]
+        public static extern int SendMessage(IntPtr hWnd, int wMsg, IntPtr wParam, StringBuilder lParam);
```

Comment:
I don't expect it to be changed by the person that calls the function but I am using this overload with the WM_GETTEXT to retrieve the title of a window/component. So the receiver will change the text, indeed.

```
        public static string GetWindowTextRaw(IntPtr hwnd)
        {
            // Allocate correct string length first
            int length = User32.SendMessage(hwnd, WM.GETTEXTLENGTH, IntPtr.Zero, IntPtr.Zero).ToInt32();

            if (length == 0)
                return string.Empty;

            StringBuilder sb = new StringBuilder(length + 1);
            if (User32.SendMessage(hwnd, WM.GETTEXT, (IntPtr)sb.Capacity, sb) == IntPtr.Zero)
            {
                return string.Empty;
            }
            else
            { 
                return sb.ToString();
            }
        }
```